pynecone/utils.py

```diff
@@ -533,6 +537,13 @@ def get_num_workers() -> int:
     return (os.cpu_count() or 1) * 2 + 1
 
 
+def get_api_port():
```

Comment:
This needs a return type annotation `-> str` as well as a doctoring to make the build happy